source/patterns/@aws-solutions-constructs/core/test/cloudfront-distribution-s3-helper.test.ts

```diff
@@ -58,7 +58,7 @@ test('check bucket metadata', () => {
         rules_to_suppress: [
           {
             id: "W35",
-            reason: "This S3 bucket is used as the access logging bucket for CloudFront Distribution"
+            reason: "This S3 bucket is used as the access logging bucket for another bucket"
```

Comment:
I don't think we want to do this. This is finding the logging bucket set up in buildS3Bucket(), which a resource by the test infrastructure and doesn't tell us anything about construct behavior. As I see it, logging should still be the default behavior, so there should still be a logging bucket with this suppression rule.

Reply:
good catch, I believe I got the `getLoggingBucket()` logic wrong, fixing...

Reply:
OK, the updated logic did not change the test result apparently. Could you elaborate as to whats going on here in terms of `rules_to_suppress` ?

Reply:
We run a tool called cfn_nag that looks for security concerns in cloudformation templates (in this case, it looks at test snapshots). It's an external tool that doesn't allow us to ignore specific rules in configuration so we need to add metadata to the CFN to remove false positives. There are 2 buckets in the construct, both with metadata suppressing W35 - one is test infrastructure ("...another bucket...") and another that is part of the construct ("...CloudFront distribution..."). So either statement should pass expect statement, but the latter is looking at the construct and evaluating what we are looking for.

Reply:
Thanks for the explanation! I change it back, not sure why I had changed it in the first place, there has been many iterations as you know 😅